src/Microsoft.AspNet.SignalR.SqlServer/DbOperation.cs

```diff
@@ -180,43 +168,28 @@ private void LoggerCommand(DbCommand command)
         [SuppressMessage("Microsoft.Usage", "CA2202:Do not dispose objects multiple times", Justification = "Disposed in async Finally block"),
          SuppressMessage("Microsoft.Reliability", "CA2000:Dispose objects before losing scope", Justification = "Disposed in async Finally block")]
 #if ASPNET50
-        private void Execute<T>(Func<IDbCommand, Task<T>> commandFunc, TaskCompletionSource<T> tcs)
+        private async void Execute<T>(Func<IDbCommand, Task<T>> commandFunc, TaskCompletionSource<T> tcs)
 #else
-        private void Execute<T>(Func<DbCommand, Task<T>> commandFunc, TaskCompletionSource<T> tcs)
+        private async void Execute<T>(Func<DbCommand, Task<T>> commandFunc, TaskCompletionSource<T> tcs)
```

Comment:
Actually I think this method now behaves differently that it did before and maybe it cannot really be `async` at all. Before if `connection.Open` had failed (which is no unlikely) the exception would have been propagated to the upstream caller. Now because the entire method is async and is invoked as fire and forget if opening the connection fails the exception will be lost and the upstream caller wouldn't even know.